src/Factory_TEST.cc

```diff
@@ -19,7 +19,7 @@
 #include <algorithm>
 #include <cstddef>
 
-#include "ignition/msgs/test_config.h"
+#include "test_config.h"
```

Comment:
I know this is still a draft, but just in case:

This upsets the code checker:

`./src/Factory_TEST.cc:22:  Include the directory when naming .h files  [build/include] [4]`

And broke all our CMake-based CI :wink: